src/react/components/searchPage.js

```diff
@@ -0,0 +1,398 @@
+          <div
+            xs={12}
+            lg={results.nbHits > 0 ? 6 : 8}
+            className={`
+              col-span-12 lg:col-span-6 lg:col-start-4
+              ${results.nbHits > 0 ? "lg:col-start-4" : "lg:col-start-3"}
+            `}
+          >
+            <SearchField
+              onInput={(queryText) => updateAutocomplete(queryText)}
+              onSubmit={(queryText) => submitSearchQuery(queryText)}
+              onReset={() => submitSearchQuery("", "reset")}
+              defaultValue={query}
+              autocompleteSuggestions={suggestions}
+              clearBtnClasses={clearBtnClasses}
+              inputClasses={inputClasses}
+              wrapperClasses={wrapperClasses}
+              submitBtnClasses={submitBtnClasses}
+              autocompleteLinkClasses={autocompleteLinkClasses}
+              autocompleteLinkFocusClasses={autocompleteLinkFocusClasses}
+              autocompleteContainerClasses={autocompleteContainerClasses}
+              clearOnEscape
+            />
+          </div>
+        </div>
+        <div className="grid grid-cols-12 lg:grid-gap mt-20 lg:mt-40">
+            {results.nbHits > 0 && (
+              <React.Fragment>
+                <div
+                  className={`col-span-12 lg:col-span-3 lg:hidden relative mb-16 ${
+                    opened ? "shadow-xl" : ""
+                  }`}
+                >
+                  <div ref={ref}>
+                    <button
+                      type="button"
+                      className={`algolia-search--filter-btn group flex w-full justify-between border px-8 py-6 text-21 font-semibold items-center transition-colors
+                        ${
+                          opened
+                            ? "opened"
+                            : "collapsed"
+                        }`}
+                      aria-expanded={opened}
+                      ref={filterOpenRef}
+                      onClick={() => setOpened(!opened)}
+                    >
+                      <span className="py-[14px] flex">
+                        {opened ? "Filters" : " Filter results"}
+                      </span>
+                      {opened ? (
+                        <span className="algolia-search--filter-close-btn ml-02em font-regular flex items-center text-18 group-hocus:underline">
+                          Close
+                          <Icon icon="x" className="w-14 ml-2" />
+                        </span>
+                      ) : (
+                        <span className="flex items-center mt-0 text-digital-red-light group-hocus:text-white hocus:shadow-none">
+                          <Icon icon="chevron-down" />
+                        </span>
+                      )}
+                    </button>
+
+                    {opened && (
+                      <div className="absolute top-[100%] left-0 w-full z-10 bg-white shadow-2xl border border-solid border-black-10">
```

Comment:
This package looks to be using the traditional mode for tailwind so classes such as `top-[100%]` won't work.

Reply:
Feel free to refactor and upgrade into JIT mode.

Reply:
Yeah i think it needs to be updated to later version of TW - I don't think JIT is in core before version 2

Reply:
Yeah - I was trying to refactor things to not use the JIT mode. I'll do another pass for any I missed like this one.